README.md

```diff
@@ -231,7 +231,6 @@ The Ditto Heartbeat tool allows you to monitor, locally or remotely, the peers i
 There are three values you need to provide to the Heartbeat:
```

Comment:
Should be 4. I'd reword to say "these are the values we need to provide to the Heartbeat config" or something like that